pyproject.toml

```diff
@@ -86,7 +83,3 @@ lint = [
 includes = ["src"]
 package-dir = "src"
 # source-includes = ["tests"]
-
-[tool.pdm.resolution.overrides]
```

Comment:
Removing this as I did not notice any issue in lock resolution with respect to these.